routes/error-tracker.js

```diff
@@ -49,6 +47,39 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  const location = '([^ \\n]+):(\\d+):(\\d+)';
+  const chromeStackTraceRegex = new RegExp(`^\\s*at (.+ )?(?:${location}|\\(${location}\\))$`, 'gm');
+  const mozillaSafariStackTraceRegex = /^([^@\n]*)@(.+):(\d+):(\d+)$/gm;
+  let validException = '';
+  let match;
+  let validExceptions = [];
+  if (chromeStackTraceRegex.test(exception)) {
+    while ((match = chromeStackTraceRegex.exec(exception))) {
```

Comment:
Exec has significantly better performance. https://jsperf.com/exec-vs-match-vs-test-vs-search/2. Given the number of requests, I thought using exec would be better.